hera_cal/io.py

```diff
@@ -2184,6 +2245,240 @@ def write_vis(fname, data, lst_array, freq_array, antpos, time_array=None, flags
         return uvd
 
 
+def create_uvd_from_hera_data(
```

Comment:
I'm confused why `write_vis` isn't using this function? Seems like they do something very similar, no?

Reply:
Also, a ton of uncovered lines in this function...

Reply:
Yeah, I'd say that `write_vis` should use this function, to keep things tidy. I'll have a crack at it.

Reply:
Ah, I don't use this any more either, so I've removed it from this PR. In general it might be a nice thing to have (sometimes you want to get a UVData object without writing a file), but again we can think about that more clearly later.